### PR TITLE
fix: value should be aligned if `min` `max` changed

### PR DESCRIPTION
**Pull request by benjycui**
Close: https://github.com/react-component/slider/issues/54

### CODE REVIEW

File: src/Slider.jsx [hunk 1]
```diff
@@ -214,10 +231,14 @@ class Slider extends React.Component {
     return precision;
   }
 
-  trimAlignValue(v) {
+  isAlignedByMinOrMax(value, props) {
```
Comment (just-boris) [inline src/Slider.jsx]: I think that `isValueOutOfBounds` is better name for function.

**Comment by just-boris** [comment]
Except the little naming issue, looks good.
@benjycui :+1:

**Comment by benjycui** [comment]
Not a native speaker.  囧

**Comment by just-boris** [comment]
@benjycui that's fine, English is not my native language too :smile:

**Comment by benjycui** [comment]
If you had reviewed this PR, and thought it is OK. Please merge it :-)

**Pull request merged**